acq4/devices/AxoPatch200/devGuiTemplate.py

```diff
@@ -24,7 +24,8 @@ def _translate(context, text, disambig):
         return QtGui.QApplication.translate(context, text, disambig)
 
 class Ui_devGui(object):
-    def setupUi(self, devGui):
```

Comment:
Oh, sure. Sorry.

I did as you suggested but the modeCombo options don't appear in the central gui once acq4 is launched. 

I also had to copy these definitions in the AxoPatch200.py file which I doubt is ideal. 

try:
    _encoding = QtGui.QApplication.UnicodeUTF8
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig, _encoding)
except AttributeError:
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig)

Do you have an idea why it is not working.